### DISCUSSION TITLE
"core:*" tags

### DISCUSSION THREAD

**Discussion by rarkins** [renovatebot/renovate#10347] [Suggest an Idea]
@HonkingGoose @viceice @JamieMagee you may have noticed I added some "core" labels recently:

![image](https://user-images.githubusercontent.com/6311784/121147250-fe9b8800-c840-11eb-99ff-783f9bcc1b61.png)

The general idea is that we have:
- managers
- versioning
- datasources
- platforms
- core functionality

And that the majority of Issues should cover at least one of those, and should also map approximately to our semantic commit scopes.

**Comment by HonkingGoose** [comment]
I'll make a PR to add this to `docs/development/issue-labeling` (see #10349).

The `core:schedule` and `core:automerge` labels don't have a description. Maybe you could add one? Or do you think it's clear enough from the label name?

Search on GitHub for `core:` labels: https://github.com/renovatebot/renovate/labels?q=core%3A

**Reply by rarkins** [reply]
I've added descriptions to be consistent, although the descriptions don't really add much!

**Reply by HonkingGoose** [reply]
Thank you! 😄

**Comment by viceice** [comment]
Sounds good.
